src/java17/groovy/eu/xenit/gradle/enterprise/conventions/extensions/dockerimagelabels/SpringBootDockerImageLabelsApplier.groovy

```diff
@@ -0,0 +1,32 @@
+package eu.xenit.gradle.enterprise.conventions.extensions.dockerimagelabels
```

Comment:
I notice `SpringBootDockerImageLabelsApplier` and `GradleDockerPluginImageLabelsApplier` use groovy, instead of java and they are the only .groovy source code files in the project.

Slighly odd to split up these source sets, if not for a particular reason ?

Reply:
Excellent reason :ok_hand: 

Maybe we should put this in a comment in the class itself ?